sites/next.skeleton.dev/src/content/docs/get-started/figma.mdx

```diff
@@ -1,26 +1,57 @@
 ---
 layout: '@layouts/LayoutDoc.astro'
 title: Figma
-description: A Figma UI Kit for Skeleton.
+description: The Figma UI Kit for Skeleton.
 order: 60
 ---
 
 export const components = componentSet;
 
-The Skeleton Figma UI Kit has been designed in coordination between [Bohdan Karplyak](https://www.linkedin.com/in/bohdan-karplyak-659a41162/), [Micha Mailänder](https://www.linkedin.com/in/micha-mail%C3%A4nder-548aba16a/), and [Skeleton Labs](https://www.skeletonlabs.co/).
+import FigmaBuyNow from '@components/docs/FigmaBuyNow.astro';
+import NavGrid from '@components/docs/NavGrid.astro';
 
-<img
-	src="https://github.com/skeletonlabs/skeleton/assets/1509726/956e6893-062e-4030-ba6a-d7b6e8acc4e8"
-	alt="figma-preview"
-	class="rounded-container"
-/>
+import { Image } from 'astro:assets';
+import ImgHero from '@images/get-started/figma/hero.webp';
+import ImgComponents from '@images/get-started/figma/components.webp';
+import ImgPlugin from '@images/get-started/figma/plugin.webp';
+import ImgTutorial from '@images/get-started/figma/tutorials.webp';
+
+<div class="relative">
```

Comment:
The first thing I want to know as a user is what this page all about. For example, see the large prominent text on the [Layouts page](https://next.skeleton.dev/docs/get-started/layouts). It would be great to have something like this. A quick suggestion:

> "Skeleton Labs has coordinated with professional designers to provide a premium Figma UI Kit. This implements all available Skeleton features and components, while also providing a dedicated theme import feature to help your designers get started quickly."

(feel free to improve as desired)